src/main/java/core/basesyntax/DrawingFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawingFigure {
+    void drawFigure();
```

Comment:
The method name `drawFigure` in the interface could be simplified to `draw` since the context of drawing is already established by the interface name `FigureDrawer` (after renaming from `DrawingFigure`).